modules/starfish/2.0/starfish2.smk

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Ryan Morin
+# Module Author:    Laura Hilton
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+import glob 
+import itertools as it
+import string
+import copy
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["starfish"]`
+CFG = op.setup_module(
+    name = "starfish",
+    version = "2.0",
+    subdirectories = ["inputs", "starfish", "outputs"],
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _starfish_input_vcf,
+    _starfish_rename_output, 
+    _starfish_output_vcf,
+    _starfish_output_union,
+    _starfish_output_venn,
+    _starfish_all,
+
+##### GLOBAL VARIABLES #####
+
+# All variant callers from config
+callers = list(CFG["inputs"]["vcf"].keys())
+callers = [caller.lower() for caller in callers]
+
+# The name of the union VCF file
+union_vcf = copy.deepcopy(callers)
+union_vcf.append("union")
+union_vcf = "_".join(union_vcf)
+not_union_vcf = "!" + union_vcf
```

Comment:
wut?

Reply:
Oh yeah I can delete the not_union_vcf variable. The rest combines the names of the variant callers and appends "union" to it, e.g. "strelka_lofreq_mutect_union"

Reply:
I get that. I just didn't understand why the not_union one existed. Was this an attempt to construct a regex?